test/runtests.jl

```diff
@@ -25,4 +25,8 @@ if already_installed
     Conda.add("curl")
 end
 
-@test isfile(joinpath(Conda.scriptsdir("conda"), "conda" * @windows ? ".exe": ""))
\ No newline at end of file
+@test isfile(joinpath(Conda.SCRIPTDIR, "conda" * @windows ? ".exe": ""))
+
+Conda.add("jupyter=4.0.0")
+@test v"4.0.0"==run(`$(joinpath(Conda.SCRIPTDIR, "jypyter")) kernelspec --version`)
```

Comment:
The old equivalent to this would be `@test v"4.0.0"==run(`$(joinpath(Conda.scriptsdir(), "jypyter")) kernelspec --version`)` which would fail because the script do not work.